ConfCore/Adapter.swift

```diff
@@ -7,46 +7,28 @@
 //
 
 import Foundation
-import SwiftyJSON
 
-enum AdapterError: Error {
-    case invalidData
-    case unsupported
-    case missingKey(JSONSubscriptType)
+// This is needed to keep the same behavior that adapters had before
+// Where it could adapt array of items even if some of the individual items failed to adapt
+struct FailableItemArrayWrapper<T: Decodable>: Decodable {
```

Comment:
From a naming perspective, this is actually a wrapper that can never fail to decode.